### ISSUE TITLE
Failed to auth with captcha

### ISSUE THREAD

**Issue by duo** [rodjunger/chatgptauth#1]
Hi

Login with capthca failed wit
```
1:32PM INF Creating auth client
1:32PM INF Starting authentication process password=<my password> username=<my email>
1:32PM INF Got main page
1:32PM INF Got CSRF token token=2d39ec46711bd0c0e4559f64b3bc914a353ccfe66fe1d673ef76005425c1eaa6
1:32PM INF Got auth0 URL url=https://auth0.openai.com/authorize?client_id=<my client id>&scope=openid%20email%20profile%20offline_access%20model.request%20model.read%20organization.read&response_type=code&redirect_uri=https%3A%2F%2Fchat.openai.com%2Fapi%2Fauth%2Fcallback%2Fauth0&audience=https%3A%2F%2Fapi.openai.com%2Fv1&prompt=login&state=vTLekskUYyPyk4W27jNfoJTqncB2u7ZULlfrYmf2uc4&code_challenge=GZOSJW_18H1RthKYYmJx3bYi7V0TuF1LkPAssn49Jtg&code_challenge_method=S256
1:32PM INF Captcha detected
1:32PM INF Got auth0 authorization hasCaptcha=true
Captcha answer: rHgtQb
1:32PM INF Username sent
1:32PM INF Password sent
1:32PM INF Session resumed
{"level":"error","error":"authCallback: couldn't find token","time":"2022-12-11T13:32:55+08:00","message":"Failed to finish auth"}
```

The nexData is `{"props":{"pageProps":{}},"page":"/auth/error","query":{},"buildId":"ShICingOrjj6GgETAxwpI","nextExport":true,"autoExport":true,"isFallback":false,"scriptLoader":[]}`

OS: Linux x86_64
Commit: 7f01406196f55a30e2472656e8e1c69b7cc0c7b4

**Comment by rodjunger** [comment]
Hi @duo, I'm pretty sure that this error is kind of a time-out, it happened to me while I was developing and even if i tried to log-in through the browser I would still get that error.

I'd recommend waiting a couple of hours and trying again or trying with a differente proxy (different proxy wasn't enough for me tho, I had to wait it out).
I just tested the log-in locally with no proxy on my machine and it worked so it's probably not related to the code or some change that they might have done.

```
4:06AM INF Password sent
4:06AM INF Session resumed
4:06AM INF Logged in token=
```

Also I'll remove the password from the logs, doesn't make a lot of sense and it will be a problem if someone forgets to remove it before posting them.
